Cargo.toml

```diff
@@ -164,19 +164,11 @@ lto = false
 # Needed for reasonable performance
 flate2.opt-level = 2
 insta.opt-level = 3
-libz-rs-sys.opt-level = 2
```

Comment:
This needs to be reverted. I have seen cargo do this sometimes but I haven't figured out what causes it.